File: beginner/c/2031.c
#include <stdio.h>
#include <string.h>

int main() {
    int i, n;
    char j1[10], j2[10];

    scanf("%d", &n);

    for(i = 0; i < n; i++){
        scanf("%s", j1);
        scanf("%s", j2);

        if(strcmp(j1, "ataque") == 0 && strcmp(j2, "ataque") == 0){
            printf("Aniquilacao mutua\n");
        } else if(strcmp(j1, "pedra") == 0 && strcmp(j2, "pedra") == 0){
            printf("Sem ganhador\n");
        } else if(strcmp(j1, "papel") == 0 && strcmp(j2, "papel") == 0){
            printf("Ambos venceram\n");
        } else if(strcmp(j1, "ataque") == 0 && strcmp(j2, "pedra") == 0){
            printf("Jogador 1 venceu\n");
        } else if(strcmp(j1, "pedra") == 0 && strcmp(j2, "ataque") == 0){
            printf("Jogador 2 venceu\n");
        } else if(strcmp(j1, "pedra") == 0 && strcmp(j2, "papel") == 0){
            printf("Jogador 1 venceu\n");
        } else if(strcmp(j1, "papel") == 0 && strcmp(j2, "pedra") == 0){
            printf("Jogador 2 venceu\n");
        } else if(strcmp(j1, "ataque") == 0 && strcmp(j2, "papel") == 0){
            printf("Jogador 1 venceu\n");
        } else if(strcmp(j1, "papel") == 0 && strcmp(j2, "ataque") == 0){
            printf("Jogador 2 venceu\n");
        } 
    }

    return 0;
}

File: beginner/java/2031.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int n = scanner.nextInt();
        scanner.nextLine(); 

        for (int i = 0; i < n; i++) {
            String j1 = scanner.nextLine();
            String j2 = scanner.nextLine();

            if (j1.equals("ataque") && j2.equals("ataque")) {
                System.out.println("Aniquilacao mutua");
            } else if (j1.equals("pedra") && j2.equals("pedra")) {
                System.out.println("Sem ganhador");
            } else if (j1.equals("papel") && j2.equals("papel")) {
                System.out.println("Ambos venceram");
            } else if (j1.equals("ataque") && j2.equals("pedra")) {
                System.out.println("Jogador 1 venceu");
            } else if (j1.equals("pedra") && j2.equals("ataque")) {
                System.out.println("Jogador 2 venceu");
            } else if (j1.equals("pedra") && j2.equals("papel")) {
                System.out.println("Jogador 1 venceu");
            } else if (j1.equals("papel") && j2.equals("pedra")) {
                System.out.println("Jogador 2 venceu");
            } else if (j1.equals("ataque") && j2.equals("papel")) {
                System.out.println("Jogador 1 venceu");
            } else if (j1.equals("papel") && j2.equals("ataque")) {
                System.out.println("Jogador 2 venceu");
            }
        }

        scanner.close();
    }
}
